scss/bootstrap/components/_time-amount-picker.scss

```diff
@@ -0,0 +1,7 @@
+.time-amount-picker {
+
+  select {
+    padding-right: 12px;
+    width: 44px;
+  }
+}
```

Comment:
I think if you are styling a single child of a element better not to nest the styles.  So, i prefer to keep it as following:
```css
.time-amount-picker select {
   padding-right: 12px;
   width: 44px;
}
```